lua-guidelines.md

```diff
@@ -66,19 +66,18 @@ need_*, should_*, must_*, in_*, not_* и т.п.
 для функций, не имеющих отношения к созданию объектов. Для таких функций
 рекомендуется префикс create_.
 
+##Кавычки
+
+Рекомендуемые кавычки по-умолчанию - апострофы. Если в строке содержатся
+апострофы - использовать для данной строки кавычки по своему усмотрению.
+
 ##Пробельные символы
 
 В коде **нельзя использовать символы табуляции** (кроме мэйкфайлов). Если в
 строковом литерале нужен символ табуляции, используйте
 
     "\t".
 
-Строки желательно ограничивать теми кавычками, которых нет в тексте:
-
-    "\"" -> '"'
-    '\'' -> "'"
-    "\"'" -> [["']]
-
```

Comment:
* Why example was removed 

  I've added separate section about quoting. It recommends apostrophes by default and in case quoted string already have apostrophes allows any quoting author feels suitable. This is weaker requirement than removed text and examples demonstrated.

  Sometimes even escaping may be more desirable than other alternatives. For example quoting string `\n\t\n\"\'`. Using long quotes is not safe for this case and breaks formatting style.

* Why apostrophes, not quotes

  This is my personal feel of cosiness.
  * effective typing
  * in quoted strings `"` occurs much more often than `'`. So no need to escape it `\"`.

* Mass changes
  
  In my codebase I do not consider such conservative arguments. Decisions based only on "I think it's more effective" or even "why not try?". If you can't make even small class of changes, you're not own this code, just maintain it. (No offence, this is my personal opinion just to state why I'm ignoring this aspect.)

As a weaker variant it may be recommended to use one quote style by _file_, not whole _codebase_.